### DISCUSSION TITLE
Clickhouse cluster for self-hosted plausible installation

### DISCUSSION THREAD

**Discussion by AndreyBienko** [plausible/analytics#4534] [Self-Hosted Support]
Good day and thanks for the great software

I am currently running a self-hosted plausible with a separate standalone clickhouse instance. 
I'm considering making the instances fail-safe.  When starting the plausible application, I noticed that it does migrations that do not assume that clickhouse will be used not in a standalone configuration. 

Is there a possibility to use clickhouse cluster together with plausible?

**Comment by ruslandoga** [comment]
👋 @AndreyBienko 

This issue might be relevant: https://github.com/plausible/analytics/issues/4454

The next release https://github.com/plausible/analytics/pull/4483 would include up-to-date SQL schema dumps https://github.com/plausible/analytics/pull/4481 so it might become easier to use the "bootstrapped" approach I mentioned in that issue.
